neuralprophet/forecaster.py

```diff
@@ -1660,22 +1670,27 @@ def plot(
                     plot_history_data=True,
                 )
 
-        # Check whether the default plotting backend is overwritten
+        # Check whether a local or global plotting backend is set.
         plotting_backend = (
-            plotting_backend
-            if plotting_backend != "default"
-            else (self.plotting_backend if hasattr(self, "plotting_backend") else "matplotlib")
+            auto_set_plotting_backend(plotting_backend)
+            if plotting_backend is not None
+            else (
+                auto_set_plotting_backend(self.plotting_backend)
+                if hasattr(self, "plotting_backend")
+                else auto_set_plotting_backend("plotly-auto")
+            )
```

Comment:
Suggestion: For ease of reading, you could use a more traditional if-else format.